app/gui/view/welcome-screen/src/template_cards.rs

```diff
@@ -0,0 +1,153 @@
+//! Template cards for Welcome Screen.
+//!
+//! Template cards allows user to select a predefined template by clicking on the corresponding
+//! card.
+
+use ensogl::prelude::*;
+
+use crate::ClickClosure;
+use crate::Frp;
+
+use ensogl::system::web;
+use ensogl::system::web::AttributeSetter;
+use ensogl::system::web::NodeInserter;
+use web_sys::Element;
+use web_sys::HtmlDivElement;
+
+
+
+// ========================
+// === Cards Definition ===
+// ========================
```

Comment:
This should be divided into card struct definition and hardcoded card instances. Please read the sling guide to learn more about how to use sections.

Reply:
Devided into two subsections. It was my initial idea, but then I decided to remove subsections because they seem strongly related to each other. There are no strict rules about it, so it's hard to choose